packages/jellyfish-wallet-mnemonic/src/mnemonic.ts

```diff
@@ -124,9 +127,22 @@ export class MnemonicHdNode implements WalletHdNode {
     const inputs: SignInputOption[] = prevouts.map(prevout => {
       return { prevout: prevout, ellipticPair: this }
     })
-    return TransactionSigner.sign(transaction, inputs, {
+
+    const signed = await TransactionSigner.sign(transaction, inputs, {
       sigHashType: SIGHASH.ALL
     })
```

Comment:
when signingCb get passed in you should await until the promise resolve before signing.